config.yaml.defaults

```diff
@@ -16,6 +16,9 @@ app:
                         # base64.b64encode(os.urandom(50)).decode('ascii')
   factory: skyportal.app_server.make_app
 
+  # this endpoint does not actually do anything -- it is just for testing
+  sedm_endpoint: http://pharos.caltech.edu/add_fritz
```

Comment:
This should be mocked out.  Let me know if you'd like examples of how that works.

Reply:
This is a mock, in that it is a dummy link that provides the API and appropriate responses but does not actually submit to the queue.

Reply:
@dannygoldstein it's not a mock (web-less simulation of a 3rd party interaction) but also I dont think a Mock is needed here: we have many endpoints in the app that are not mocked right now. The tests could be made to fail gracefully if the endpoint is done. @stefanv what do you think? IMO this is ok as is.

Reply:
Given that this is a stable endpoint, I'm fine with it as is.